### DISCUSSION TITLE
How do I add an embedded circuit?

### DISCUSSION THREAD

**Discussion by ghost** [hneemann/Digital#585] [Q&A] [answered]
It's supposed to be `Components => Custom` but that doesn't exist on my version. (Latest release, Windows 10) How else can I add a circuit from another file? 

![image](https://user-images.githubusercontent.com/55333787/102719905-99110a80-42a5-11eb-8c54-f92947a67c0a.png)

**Comment by hneemann** [comment, marked as answer]
You have to save the circuit. 
After that, all the circuits that are in the same folder will appear in this menu.

**Reply by ghost** [reply]
It works, thank you! That solves my problem, maybe it could be added to the documentation for others if they have the same problem :)

**Reply by FonkinS** [reply]
But where in the menu would this go?

**Reply by FonkinS** [reply]
Reading the documentation it says as long as the embedded and final circuit are in the same folder it should work, but it doesn't seem to work
<img width="766" alt="Screenshot 2021-12-22 at 13 44 11" src="https://user-images.githubusercontent.com/85748962/147095241-cdfcf193-3f80-4a2b-94a6-e47d00211434.png">
<img width="1022" alt="Screenshot 2021-12-22 at 13 44 34" src="https://user-images.githubusercontent.com/85748962/147095249-e8307c0a-06fc-44f6-ae36-469d063c0a95.png">
